src/templates/ucp.html

```diff
@@ -350,9 +404,67 @@ <h3>{{ 'Experiment Templates'|trans }}</h3>
             </div>
           </form>
           {% include('steps-links-edit.html') %}
+        {% else %}
+        <div class="badgetab">
+          <h5 style="display: inline;">{{ template.name }}</h5>
+          <span class="tpl-noteditable"><i>This template is not editable</i></span>
```

Comment:
don't add a "tpl-noteditable" css class but use something like https://github.com/elabftw/elabftw/blob/master/src/templates/view-edit.html#L24

so basically add `|msg('ok', false')`

Reply:
Removed this class in the recent commit.